firmware/integration/rusefi_config.txt

```diff
@@ -829,12 +829,12 @@ custom maf_sensor_type_e 4 bits, S32, @OFFSET@, [0:1], @@maf_sensor_type_e_enum@
 	bit showHumanReadableWarning
 	bit stftIgnoreErrorMagnitude;+If enabled, adjust at a constant rate instead of a rate proportional to the current lambda error.  This mode may be easier to tune, and more tolerant of sensor noise.  Use of this mode is required if you have a narrowband O2 sensor.;
 	bit dcMotorIdleValve;+Used on some German vehicles around late 90s: cable-operated throttle and DC motor idle air valve.\nSet the primary TPS to the cable-operated throttle's sensor\nSet the secondary TPS to the mini ETB's position sensor(s). 
-	bit unusedBit_251_12
-	bit unusedBit_251_13
-	bit unusedBit_251_14
-	bit unusedBit_251_15
-	bit unusedBit_251_16
-	bit unusedBit_251_17
+	brain_pin_e can2TxPin;set_can2_tx_pin X
```

Comment:
I do not get this. you are adding pins in the middle of existing bitfield, this would mess up all offsets?

Reply:
what's the size of `brain_pin_e`?

Reply:
@ksmola it's 1 byte but my focus is the part where you add it into a 32 bit bitset, or so it looks like?

```
custom brain_pin_e 1 bits,    U08,   @OFFSET@, [0:7], @@brain_pin_e_enum@@